fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/mixin/client/keybinding/MixinKeyBindingAccessor.java

```diff
@@ -0,0 +1,13 @@
+package net.fabricmc.fabric.mixin.client.keybinding;
+
+import net.minecraft.client.options.KeyBinding;
+import net.minecraft.client.util.InputUtil;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(KeyBinding.class)
+public interface MixinKeyBindingAccessor
+{
+	@Accessor("keyCode")
+	InputUtil.KeyCode fabric_getConfiguredKeyCode();
```

Comment:
@asie/modmuss/whoever, would it be better to implement this interface on the existing MixinKeyBinding rather than have a separate accessor mixin?

Reply:
@Prospector I would definitely say yes since accessor mixins are known to be buggy a lot.

Reply:
If you find bugs with accessors and invokers you should report them. They tend to only break if you use them incorrectly, or if the LVT messes up in an injection callback, which is unrelated to the accessor/invoker itself.

Reply:
well the question still remains whether this should be 1 or 2 mixins

Reply:
A separate mixin is best imo